### ISSUE TITLE
When Storage heart used, next time the game is saved it crashes

### ISSUE THREAD

**Issue by ghost** [blushiemagic/MagicStorage#73]
even with only magic storage and cheat sheet, every time i use the storage heart, the next time the game saves, whether it be automatic or manual, it will crash, deleting everything in the storage system and the storage blocks

**Comment by gwentoo** [comment]
I had the same problem, try using the normal version of tmodloader instead of the beta versions, worked for me.

**Comment by HarriXS** [comment]
Yeah, I ended up finishing my playthrough and decided to see if tMod v10 worked with it and it did :)

**Comment by ChromeeMC** [comment]
same issue

**Comment by absoluteAquarian** [comment]
The following message has been copy/pasted to all issues older than 3 months.

In order to clean up how the project looks, I'll be closing all old issues.  If you believe that this issue is still relevant, please make a new issue that references this issue in its message.
